engines/scumm/imuse/imuse.cpp

```diff
@@ -1342,6 +1350,40 @@ void IMuseInternal::update_volumes() {
 	}
 }
 
+void IMuseInternal::musicVolumeReduction(MidiDriver *midi) {
+	int curVol;
+	int curEffVol;
+	int factor = 2; // The music volume variables are 0-255, and we need 0-127
+
+	if (_paused)
+		return;
+
+	_musicVolumeReductionTimer += midi->getBaseTempo();
+	while (_musicVolumeReductionTimer >= 16667) {
```

Comment:
Perhaps it would be better to use a define for this, e.g.:

  #define VOLUME_REDUCTION_RATE    16667  // 16.667 ms per tick

Reply:
Yes, that's a good idea. I should probably do that as well on the `dimuse` source file.

I should wait for athrxx to confirm (or even non-deny) that this is the correct rate for all the games.

Reply:
I am pretty sure that MI2 and INDY4 don't have a volume reduction mechanism like that, since there never was a MI2 talkie version, and the INDY4 talkie was published much later than the     floppy version. And the IMS drivers are almost identical for MI2 and INDY4. DOTT has slightly modernized drivers, but not anywhere like the new system used in SAMNMAX. I doubt that DOTT would use volume reduction like that, but I haven't checked yet. So, if you do want to extend this to all games anyway, it is probably okay to use the timing from SAMNMAX, too.

Reply:
...you are right. Just had a proper Streisand effect 😛 This volume reduction is completely absent on DOTT and below (I extended it on MI1 and MI2 just because I thought it was on INDY4 and its same engine was used on the Ultimate Talkies).

This is easier than expected then. I'll just limit this to SAMNMAX. Thanks!